packages/foreman/tfm/tfm.spec

```diff
@@ -21,7 +19,7 @@
 Summary: Package that installs %scl
 Name: %scl_name
 Version: 5.0
-Release: 11%{?dist}
+Release: 12%{?dist}
```

Comment:
I think dropping tfm-ror52 is enough to warrant a full version bump

Reply:
Thought that might be a thing, but good call!